content/en/docs/how-tos/rbac.md

```diff
@@ -44,6 +44,57 @@ No API is provided to create Rover groups. We have to do some manual work on the
 The existing groups defined in the release repo will be removed from the release repo and the clusters. 
 If such a group is used as a subject in a RBAC manifest, then its owner has to ensure the group also exists on Rover. In case that the group name cannot be managed by the owner, e.g., the desired group name is taken by other teams, then the RBAC manifest has to use another Rover group.
 
+For example, `group/cvp-pool-admins` cannot exist any more in the release repo because it contains usernames.
+
+```yaml
+apiVersion: v1
+kind: List
+items:
+- apiVersion: user.openshift.io/v1
+  kind: Group
+  metadata:
+    name: cvp-pool-admins
+  users:
+  - bob
+  - jim
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: RoleBinding
+  metadata:
+    name: cvp-pool-admins
+    namespace: cvp-cluster-pool
+  roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: ClusterRole
+    name: hive-cluster-pool-admin
+  subjects:
+  - apiGroup: rbac.authorization.k8s.io
+    kind: Group
+    name: cvp-pool-admins
+```
+
+When it is removed from the release repo and the cluster, the members of the group lose the corresponding permissions on the clusters.
+To retrieve the permissions, we need
+
+- create a group `cvp-pool-admins` at [Rover](https://rover.redhat.com/groups/) with the same members, and
+
+- update the config file with the following stanza because the group is needed only on the `hive` cluster.
+
+```yaml
+groups:
+  cvp-pool-admins:
+    clusters:
+    - hive
+```
+
+If another Rover group name is referred, we can do the group mapping by the following configuration:
```

Comment:
Explain if the bindings need to change - will the Rover sync create a k8s `Group` with the same name>

Reply:
Updated.